src/lib/email-service.js

```diff
@@ -0,0 +1,47 @@
+const nodemailer = require("nodemailer")
+const config = require('../config')
+
+
+const devConfig = {
```

Comment:
since we're using the config style now, defining the configs here will make it require all the dev and prod at once. that is when we start calling the library, it will throw errors for both production and development envs though we don't need them all depending on the ienvironment

i think its better to use one variable and then only assign in an if statement 

something like 
```
let config;
if(config('NODE_ENV') === 'production'){
   config = ...
}
else{
 config = ...
}
```